pkg/fs/fs.go

```diff
@@ -762,7 +764,9 @@ func (fs *FileSystem) doResolve(ctx meta.Context, p string, followLastSymlink bo
 		if len(name) == 0 {
 			continue
 		}
-		if parent == meta.RootInode && i == len(ss)-1 && vfs.IsSpecialName(name) {
+		isWinSpCtrlFile := i == len(ss)-1 && runtime.GOOS == "windows" && vfs.IsSpecialControlName(name)
```

Comment:
we can run s3 gateway or Java SDK on Windows, so they are still affected.

Reply:
The related logic has been moved to winfs.getattr and winfs.open now.